pytorch_lightning/utilities/apply_func.py

```diff
@@ -77,7 +77,6 @@ def apply_to_collection(
 ) -> Any:
     """
     Recursively applies a function to all elements of a certain dtype.
-
```

Comment:
@tchaton it's really essential that we keep these empty lines for proper rendering of docs

Reply:
Yes, sounds good.